Fall-2017-JS02/Challenge2/project.js

```diff
@@ -1 +1 @@
-console.log("Hi there!")
+console.log("Hi there") //making the console state what is inside the quotes/
```

Comment:
I like comments that are more of human dialogue, `making the console tell us a greeting`